WordPress/Classes/Networking/AccountServiceRemoteREST.m

```diff
@@ -11,12 +18,12 @@ @implementation AccountServiceRemoteREST
 
 - (id)initWithApi:(WordPressComApi *)api
```

Comment:
Nm. I saw somewhere else where a service's init method was implemented with `instancetype` and its declaration used `id` so making it here bothers me less :)